.github/actions/run_tests/action.yml

```diff
@@ -1,24 +0,0 @@
-name: 'Run tests'
-description: 'This composite action runs the tests'
-runs:
-  using: "composite"
-  steps:
-    - name: Build Poetry
-      run: poetry build
-      shell: bash
-
-    - name: Install project whl
-      run: pip install dist/*.whl
-      shell: bash
-
-    - name: Run Python unit tests
-      run: poetry run nox -s unit_tests
-      shell: bash
```

Comment:
unit tests we still want to run in github for quicker feedback without logging in into AWS